metpy/plots/__init__.py

```diff
@@ -6,10 +6,14 @@
 # Trigger matplotlib wrappers
 from . import _mpl  # noqa: F401
 from ._util import add_metpy_logo, add_timestamp, add_unidata_logo
+from .ctables import *  # noqa: F403
 from .skewt import *  # noqa: F403
 from .station_plot import *  # noqa: F403
+from .wx_symbols import *  # noqa: F403
 
 __all__ = skewt.__all__[:]  # pylint: disable=undefined-variable
 __all__.extend(station_plot.__all__)  # pylint: disable=undefined-variable
+__all__.extend(ctables.__all__)  # pylint: disable=undefined-variable
```

Comment:
Can you put `ctables` above `skewt` to keep them alphabetized/match import order?